flink-queryable-state/flink-queryable-state-client-java/src/main/java/org/apache/flink/queryablestate/network/AbstractServerHandler.java

```diff
@@ -262,7 +262,7 @@ public void run() {
 					try {
 						stats.reportFailedRequest();
 
-						final String errMsg = "Failed request " + requestId + ". Caused by: " + ExceptionUtils.stringifyException(t);
+						final String errMsg = "Failed request " + requestId + ".\nCaused by: " + ExceptionUtils.stringifyException(t);
```

Comment:
Why is there a newline? I think this can look strange in a log.

Reply:
I put it there because it looked actually nicer. But I suppose this is a matter of personal taste.

Reply:
this should be `System.lineSeparator()`